python/langsmith/_internal/otel/_otel_exporter.py

```diff
@@ -176,6 +194,7 @@ def export_batch(
                     )
                     if span:
                         self._spans[op.id] = span
+                        self._span_timestamps[op.id] = time.time()
```

Comment:
Can we just have these share a single dict for simplicity?

`self._span_info = { "span": span, "created_at": time.time() }`